GameServerLib/Content/CharData.cs

```diff
@@ -158,12 +170,108 @@ public void Load(string name)
                 ExtraSpells[i] = file.GetString("Data", $"ExtraSpell{i + 1}", ExtraSpells[i]);
```

Comment:
Seems like `ExtraSpells` get assigned twice (in some sort of recursive way)? Why is that?

Reply:
Thought I replaced that, will change.